### DISCUSSION TITLE
changing button severity colors with tailwind

### DISCUSSION THREAD

**Discussion by rependa** [primefaces/community#355] [PrimeVue] [answered]
Maybe there's a simple answer to this...
I'm using PrimeVue (unstyled) and Tailwind very similar to the stackblitz example on the button page in the Theme tab: https://stackblitz.com/edit/w2xkbn-qogkvj?file=src%2Fmain.js,src%2FApp.vue.

How would I go about theming the different severity background colors?

**Comment by tugcekucukoglu** [comment, marked as answer]
You could override the Tailwind presets or directly change the Tailwind presets itself.

**Reply by rependa** [reply]
Thanks for taking the time to respond. I've been trying to do exactly that, but am having a tough time getting it to work. Would you be so kind as to provide me with an example of what you're describing?
